.github/pull_request_template.md

```diff
@@ -1,3 +1,33 @@
+## For Internal Contributors
+
+* Follow trunk workflow(https://trunkbaseddevelopment.com/)
+* Branch naming -> `{type}/TICKET_ID/description`
+  * Where type = `feat` / `feature` / `fix` / `chore` / `doc` / `release`
+* Always recive PR review approvals
+* Rebase your branch with main, wait for CI to pass
+* Squash merge your commit
+  * Follow the following template for both PR description and squashing to main
+  * Use imperative language in title and description
+  * The template is very similar to conventional commits. [Read More](https://www.conventionalcommits.org/en/v1.0.0/)
+
+Template
```

Comment:
A lot of thought leaders advocate for 50/72 formatting
ie: max 50 chars for PR title, and 72 for summary. (not a hard limit)
Its good to follow those ~ especially for people primarily using command line ->
max 50 chars makes these jobs more easy to do through command line:
* Rebasing
* Cherry-picking
* git log --oneline
* Git lens in your ide
 
https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html